tests/unittests/data/lib.yml

```diff
@@ -143,23 +143,23 @@ library:
           scenario-dependent: true
       variables:
         - name: generation
-          lower_bound: 0
-          upper_bound: nb_units_max * p_max
+          lower-bound: 0
+          upper-bound: nb_units_max * p_max
```

Comment:
Was that a typo ? Did the syntax change ?

Reply:
yes that is a typo if you look at the beginning of the document it was correctly written